src/api/user.ts

```diff
@@ -0,0 +1,43 @@
+import { instance } from '@/libs/api';
+import { SignFormValuesType } from '@/types/SignFormValue';
+import axios, { AxiosError } from 'axios';
+
+export const handleLogin = async (data: SignFormValuesType) => {
+  try {
+    const result = await axios.post('/api/login', data);
```

Comment:
이거 instance 로 해야되는 거 아닌가요 ??

```suggestion
  try {
    const result = await instance.post('/api/login', data);
```

Reply:
저 api는 코드잇 api에 요청보내는게 아니라 baseurl이 달라서 만든 instance로 사용 안했어요!